docs/src/literate-tutorials/linear_elasticity.jl

```diff
@@ -1,3 +1,243 @@
-# # Linear elasticity
+# # [Linear elasticity](@id tutorial-linear-elasticity)
 #
-# TBW
+# ![](linear_elasticity.png)
+#
+# *Figure 1*: Linear elastically deformed Ferrite logo.
+#
+#md # !!! tip
+#md #     This example is also available as a Jupyter notebook:
+#md #     [`linear_elasticity.ipynb`](@__NBVIEWER_ROOT_URL__/examples/linear_elasticity.ipynb).
+#-
+#
+# ## Introduction
+#
+# The classical first finite element problem to solve in solid mechanics is a linear balance
+# of momentum problem. We will use this to introduce a vector valued field, as well as the
+# [`Tensors.jl`](https://github.com/Ferrite-FEM/Tensors.jl) toolbox.
+#
+# The strong form of the balance of momentum is given by
+# ```math
+#  -\boldsymbol{\sigma} \cdot \boldsymbol{\nabla} = \boldsymbol{b}  \quad \textbf{x} \in \Omega,
+# ```
+# where $\boldsymbol{\sigma}$ is the stress tensor, $\boldsymbol{b}$ is the body force and
+# $\Omega$ the domain.
+#
+# In this example, we use linear elasticity, such that
+# ```math
+# \boldsymbol{\sigma} = \boldsymbol{E} : \boldsymbol \varepsilon
+# ```
+# where $\boldsymbol{E}$ is the elastic stiffness tensor and $\boldsymbol{\varepsilon}$ is
+# the small strain tensor that is computed from the displacement field $\boldsymbol{u}$ as
+# ```math
+# \boldsymbol{\varepsilon} = \frac{1}{2} \left(
+#   \boldsymbol{\nabla} \otimes \boldsymbol{u}
+#   +
+#   \boldsymbol{u} \otimes \boldsymbol{\nabla}
+# \right)
+# ```
+#
+# The resulting weak form is given given as follows: Find ``\boldsymbol{u} \in \mathbb{U}`` such that
+# ```math
+# \int_\Omega 
+#   \boldsymbol{\sigma} : \left(\delta \boldsymbol{u} \otimes \boldsymbol{\nabla} \right)
```

Comment:
Testfunction to the left, please :)

Reply:
I'm not following either, why do you want the test function to the left?

Reply:
It's the way.